rollup/dist.config.js

```diff
@@ -1,28 +1,36 @@
-import resolve from 'rollup-plugin-node-resolve'
-import babel from 'rollup-plugin-babel'
-import strip from '@rollup/plugin-strip'
-import ts from "rollup-plugin-ts";
+import config from 'live-connect-common/rollup.common'
 import cleaner from 'rollup-plugin-cleaner'
+import ts from '@rollup/plugin-typescript'
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import strip from '@rollup/plugin-strip'
+import babel from '@rollup/plugin-babel'
+import terser from '@rollup/plugin-terser'
 import mjsEntry from 'rollup-plugin-mjs-entry'
-import commonjs from 'rollup-plugin-commonjs';
-
 const OUTPUT_DIR = './dist'
 
 export default {
-  input: './src/index.ts',
+  ...config,
   output: [
     {
-      file: `${OUTPUT_DIR}/index.js`,
-      format: 'cjs'
+      file: `${OUTPUT_DIR}/index.cjs`,
+      format: 'cjs',
+      sourcemap: false
```

Comment:
Turn the source map on for `live-connect`. Make sure it is off for `liveconnectbuilder`.